pkg/projects/Microsoft.NETCore.DotNetHost/win/Microsoft.NETCore.DotNetAppHost.pkgproj

```diff
@@ -21,7 +21,7 @@
     <File Include="@(ArchitectureSpecificNativeFile)">
       <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
     </File>
-    <File Include="$(ProjectDir)/version.txt" />
+    <File Include="$(ProjectDir)pkg/version.txt" />
```

Comment:
Can we make all these use `$(MSBuildProjectDirectory)` instead?  That way we don't hard-code `pkg` into all these proj files, and the allow them to be moved easier.